### ISSUE TITLE
Final SVG file does not show data matrix code

### ISSUE THREAD

**Issue by myorbit** [ticktronaut/schablone#5]
The final svg doesn't include the data matrix (and also no yellow rectangle from the template). It is just blank (white).

Important to note this is currently only working if using the latest svglue from source. I tested this with svglue-0.2.2.dev1 on Python 3.5.2 and Python 3.6.0. Future requirements for svglue are now reflected in setup.py (see here: 3b86b961a35a458312d425739d41d32f5827a951).

**Comment by ticktronaut** [comment]
Until the latest changes of svglue will be pushed from github to PyPi I recommend to install svglue from source. (Version of svglue on github is 0.2.2 and the version on PyPi is 0.2.1)

This issue results from python3 support of older versions of svglue. They have been solved in the latest commits on github.
 
I documented the correct installation in schablone's README.rst:

$ pip install git+https://github.com/mbr/svglue.git@master

**Comment by myorbit** [comment]
At first this was an issue with the embedded png (broken link), which was solved with 75935e653dca371651404c6aa7d97dfca46b1e7a.
After that it still appeared to me not showing the data matrix code. I tracked this down to a bug in my file explorer and its image viewer respectively. I tried different viewers like a browser (Firefox, etc.) and finally it reveal its content to me. :)
Thanks for this awesome package!
